.github/issue_templates/Algorithim_development.md

```diff
@@ -0,0 +1,26 @@
+### Algorithm Description:
+
+### Requirements:
+
+> **Specify: (optional)**
+
+### Algorithm Code Info: Data/Input/Output
+
+> **Specify details:**
+
+### Code:
+
+```python
+# Code
+```
+
+### Requirements/ Dependencies:
+
+- [ ] 
+    1. Yes
+- [ ] 
+    2. No
+
+### If yes, specify:
+
+### Other Notes:
```

Comment:
I'd like to see a section for related issues as well, to encourage people to link them together.